<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormClienteBase</class>
 <widget class="QWidget" name="FormClienteBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_23">
     <item>
      <widget class="QLabel" name="LNumCliente">
       <property name="text">
        <string>Numero de Cliente:</string>
       </property>
       <property name="buddy">
        <cstring>LENumCliente</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LENumCliente">
       <property name="styleSheet">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="LRazonSocial">
          <property name="text">
           <string>Razon Social:</string>
          </property>
          <property name="buddy">
           <cstring>LERazonSocial</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LERazonSocial"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Nombre:</string>
          </property>
          <property name="buddy">
           <cstring>LENombre</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LENombre"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Apellidos:</string>
          </property>
          <property name="buddy">
           <cstring>LEApellido</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEApellido"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Dirección</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Calle:</string>
          </property>
          <property name="buddy">
           <cstring>LECalle</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LECalle"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Numero:</string>
            </property>
            <property name="buddy">
             <cstring>LENumero</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LENumero"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Piso:</string>
            </property>
            <property name="buddy">
             <cstring>LEPiso</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LEPiso"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Depto:</string>
            </property>
            <property name="buddy">
             <cstring>LEDepto</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LEDepto"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Ciudad:</string>
            </property>
            <property name="buddy">
             <cstring>LECiudad</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LECiudad"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Codigo Postal:</string>
            </property>
            <property name="buddy">
             <cstring>LECodPostal</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LECodPostal"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Provincia:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ECBProvincias" name="CBProvincia">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Pais:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ECBPaises" name="CBPaises">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Información fiscal</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>CUIT/CUIL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ELECuitCuil" name="LECUITCUIL">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="inputMask">
           <string>99-99999999-9; </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Categoría de Inscripción:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CBEstadoFiscal">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_22">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Telefonos</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Telefono Fijo:</string>
            </property>
            <property name="buddy">
             <cstring>LETelFijo</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LETelFijo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Celular:</string>
            </property>
            <property name="buddy">
             <cstring>LETelCel</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LETelCel"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Fax:</string>
            </property>
            <property name="buddy">
             <cstring>LEFax</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LEFax"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Otros</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Em@il:</string>
            </property>
            <property name="buddy">
             <cstring>LEEmail</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LEEmail"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="CkBComprobanteEmail">
          <property name="text">
           <string>Enviar Comprobantes por email</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ELECuitCuil</class>
   <extends>QLineEdit</extends>
   <header location="global">ELECuitCuil.h</header>
  </customwidget>
  <customwidget>
   <class>ECBProvincias</class>
   <extends>QComboBox</extends>
   <header>ecbprovincias.h</header>
  </customwidget>
  <customwidget>
   <class>ECBPaises</class>
   <extends>QComboBox</extends>
   <header>ecbpaises.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>LERazonSocial</tabstop>
  <tabstop>LEApellido</tabstop>
  <tabstop>LENombre</tabstop>
  <tabstop>LECalle</tabstop>
  <tabstop>LENumero</tabstop>
  <tabstop>LEPiso</tabstop>
  <tabstop>LEDepto</tabstop>
  <tabstop>LECiudad</tabstop>
  <tabstop>LECodPostal</tabstop>
  <tabstop>CBProvincia</tabstop>
  <tabstop>CBPaises</tabstop>
  <tabstop>LECUITCUIL</tabstop>
  <tabstop>CBEstadoFiscal</tabstop>
  <tabstop>LETelFijo</tabstop>
  <tabstop>LETelCel</tabstop>
  <tabstop>LEFax</tabstop>
  <tabstop>LEEmail</tabstop>
  <tabstop>CkBComprobanteEmail</tabstop>
  <tabstop>LENumCliente</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
